docs/ml-features.md

```diff
@@ -251,11 +251,12 @@ frequently and don't carry as much meaning.
 `StopWordsRemover` takes as input a sequence of strings (e.g. the output
 of a [Tokenizer](ml-features.html#tokenizer)) and drops all the stop
 words from the input sequences. The list of stopwords is specified by
-the `stopWords` parameter.  We provide [a list of stop
-words](http://ir.dcs.gla.ac.uk/resources/linguistic_utils/stop_words) by
-default, accessible by calling `getStopWords` on a newly instantiated
-`StopWordsRemover` instance. A boolean parameter `caseSensitive` indicates
-if the matches should be case sensitive (false by default).
+the `stopWords` parameter. Default stop words for some languages are provided
+("danish", "dutch", "english", "finnish", "french", "german", "hungarian", "italian",
```

Comment:
I'm afraid it will confuse users as currently it does not support `StopWordsRemover.loadDefaultStopWords("English")` (with Capital E).  Maybe we should use `language.toLower` in `loadDefaultStopWords`.